tokio-util/src/net/unix/mod.rs

```diff
@@ -0,0 +1,18 @@
+//! Unix domain socket helpers.
+
+use super::Listener;
+use std::io::Result;
+use std::task::{Context, Poll};
+
+impl Listener for tokio::net::UnixListener {
+    type Io = tokio::net::UnixStream;
+    type Addr = tokio::net::unix::SocketAddr;
+
+    fn poll_accept(&self, cx: &mut Context<'_>) -> Poll<Result<(Self::Io, Self::Addr)>> {
+        Self::poll_accept(self, cx)
+    }
+
+    fn local_addr(&self) -> Result<Self::Addr> {
+        self.local_addr().map(Into::into)
+    }
+}
```

Comment:
Just put this in the other folder.

Reply:
This will force me to add `#[cfg(unix)]`.

Reply:
Ok, I don't mind whether it is separate or together then.

Reply:
Ok I'll keep it then but I don't mind either way either